configs/defconfig.esp32s2

```diff
@@ -0,0 +1,77 @@
+CONFIG_BOOTLOADER_LOG_LEVEL_NONE=y
+CONFIG_BOOTLOADER_SKIP_VALIDATE_IN_DEEP_SLEEP=y
+CONFIG_ESPTOOLPY_FLASHSIZE_4MB=y
+CONFIG_ARDUINO_RUN_CORE0=y
+CONFIG_ARDUINO_EVENT_RUN_CORE0=y
+CONFIG_ARDUHAL_ESP_LOG=y
+CONFIG_TINYUSB_MSC_BUFSIZE=4096
+CONFIG_TINYUSB_DESC_DFU_RT_STRING="Espressif DFU Device"
+CONFIG_COMPILER_OPTIMIZATION_SIZE=y
+CONFIG_COMPILER_CXX_EXCEPTIONS=y
+CONFIG_COMPILER_STACK_CHECK_MODE_NORM=y
+CONFIG_COMPILER_WARN_WRITE_STRINGS=y
+# CONFIG_SPI_MASTER_ISR_IN_IRAM is not set
+# CONFIG_SPI_SLAVE_ISR_IN_IRAM is not set
+CONFIG_ESP32S2_DEFAULT_CPU_FREQ_240=y
+CONFIG_ESP32S2_SPIRAM_SUPPORT=y
+CONFIG_SPIRAM_MALLOC_ALWAYSINTERNAL=4096
+CONFIG_SPIRAM_TRY_ALLOCATE_WIFI_LWIP=y
+CONFIG_SPIRAM_MALLOC_RESERVE_INTERNAL=0
+CONFIG_ESP32S2_KEEP_USB_ALIVE=y
+CONFIG_ETH_SPI_ETHERNET_DM9051=y
+CONFIG_ETH_SPI_ETHERNET_W5500=y
+CONFIG_ESP_HTTP_CLIENT_ENABLE_BASIC_AUTH=y
+CONFIG_HTTPD_MAX_REQ_HDR_LEN=1024
+CONFIG_HTTPD_WS_SUPPORT=y
+CONFIG_ESP_HTTPS_SERVER_ENABLE=y
+CONFIG_ESP_IPC_TASK_STACK_SIZE=1024
+CONFIG_ESP_SYSTEM_EVENT_TASK_STACK_SIZE=2048
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=4096
+CONFIG_ESP_INT_WDT_TIMEOUT_MS=1000
+CONFIG_ESP_TASK_WDT_PANIC=y
+# CONFIG_ESP_TASK_WDT_CHECK_IDLE_TASK_CPU0 is not set
+CONFIG_ESP_TIMER_TASK_STACK_SIZE=4096
+CONFIG_ESP32_WIFI_STATIC_RX_BUFFER_NUM=8
+CONFIG_ESP32_WIFI_STATIC_TX_BUFFER_NUM=8
+CONFIG_ESP32_WIFI_CACHE_TX_BUFFER_NUM=16
+CONFIG_ESP32_WIFI_CSI_ENABLED=y
+# CONFIG_ESP32_WIFI_IRAM_OPT is not set
+# CONFIG_ESP32_WIFI_RX_IRAM_OPT is not set
+# CONFIG_ESP32_WIFI_ENABLE_WPA3_SAE is not set
+CONFIG_ESP_WIFI_FTM_ENABLE=y
+CONFIG_FATFS_CODEPAGE_850=y
+CONFIG_FATFS_LFN_STACK=y
+# CONFIG_FMB_CONTROLLER_SLAVE_ID_SUPPORT is not set
+CONFIG_FMB_TIMER_PORT_ENABLED=y
+CONFIG_FREERTOS_HZ=1000
+# CONFIG_FREERTOS_ASSERT_ON_UNTESTED_FUNCTION is not set
+CONFIG_FREERTOS_WATCHPOINT_END_OF_STACK=y
+CONFIG_FREERTOS_IDLE_TASK_STACKSIZE=1024
+CONFIG_HEAP_POISONING_LIGHT=y
+CONFIG_LOG_DEFAULT_LEVEL_ERROR=y
+# CONFIG_LOG_COLORS is not set
+CONFIG_LWIP_MAX_SOCKETS=16
+CONFIG_LWIP_ETHARP_TRUST_IP_MAC=y
+# CONFIG_LWIP_DHCP_DOES_ARP_CHECK is not set
+CONFIG_LWIP_DHCP_RESTORE_LAST_IP=y
+CONFIG_LWIP_DHCP_OPTIONS_LEN=128
+CONFIG_LWIP_TCP_SYNMAXRTX=6
+CONFIG_LWIP_TCP_MSS=1436
+CONFIG_LWIP_TCP_RTO_TIME=3000
+CONFIG_LWIP_TCPIP_TASK_STACK_SIZE=2560
+CONFIG_LWIP_TCPIP_TASK_AFFINITY_CPU0=y
+CONFIG_LWIP_SNTP_MAX_SERVERS=3
+CONFIG_LWIP_DHCP_GET_NTP_SRV=y
+# CONFIG_MBEDTLS_ASYMMETRIC_CONTENT_LEN is not set
+# CONFIG_MBEDTLS_HARDWARE_MPI is not set
+# CONFIG_MBEDTLS_HARDWARE_SHA is not set
```

Comment:
These 2 should be enabled, i think: CONFIG_MBEDTLS_HARDWARE_MPI & CONFIG_MBEDTLS_HARDWARE_SHA